Lib/test/test_capi/test_complex.py

```diff
@@ -226,7 +226,10 @@ def test_py_c_pow(self):
 
         self.assertEqual(_py_c_pow(0j, -1)[1], errno.EDOM)
         self.assertEqual(_py_c_pow(0j, 1j)[1], errno.EDOM)
-        self.assertEqual(_py_c_pow(*[DBL_MAX+1j]*2)[0], complex(*[INF]*2))
+        self.assertEqual(_py_c_pow(*[DBL_MAX+1j]*2),
+                         (complex(*[INF]*2), errno.ERANGE))
+        self.assertEqual(_py_c_pow(DBL_MAX+1j, 2),
+                         (complex(*[INF]*2), errno.ERANGE))
```

Comment:
The syntax is a little bit surprising. I suggest something more regular:

```suggestion
        num = DBL_MAX+1j
        self.assertEqual(_py_c_pow(num, num),
                         (complex(INF, INF), errno.ERANGE))
        self.assertEqual(_py_c_pow(num, 2),
                         (complex(INF, INF), errno.ERANGE))
```